README.md

```diff
@@ -45,8 +47,8 @@ docker run --privileged --shm-size=1000gb --gpus all -it --rm -v ~/torch-profile
 
 Then I can just run the script:
 
-```
-python torch_prof.py
+```python3
```

Comment:
this isn't python3-formatted code, it's bash. Adding this header will apply incorrect syntax formatting.